Sophie_wdl/pilon_practice.wdl

```diff
@@ -0,0 +1,33 @@
+version 1.0
+
+workflow pilon_practice {
+  input {
+    File reference
+    File bam_file
+    File bai_file
```

Comment:
You could maybe infer this one automatically from the bam_file? (in general it's just bam_file + ".bai")